internal/snmp/translate.go

```diff
@@ -148,11 +148,14 @@ func SnmpTranslateCall(oid string) (mibName string, oidNum string, oidText strin
 	if strings.ContainsAny(oid, "::") {
 		// split given oid
 		// for example RFC1213-MIB::sysUpTime.0
-		s := strings.Split(oid, "::")
+		s := strings.SplitN(oid, "::", 2)
 		// node becomes sysUpTime.0
+		if s[1] == "" {
+			return "", oid, oid, oid, fmt.Errorf("cannot parse %v\n", oid)
+		}
 		node := s[1]
```

Comment:
```suggestion
		// node becomes sysUpTime.0
		node := s[1]
		if node == "" {
			return "", oid, oid, oid, fmt.Errorf("cannot parse %v\n", oid)
		}
```

Reply:
I like checking s[1] before assigning as there is no point assigning node just to error out.